### PR TITLE
Add nerd count to env

### PR DESCRIPTION
**Pull request by AnthonyReid99**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ module.exports = function(options) {
     environment.app_host = config.app_host; //may or may not be set. overrides the default
     environment.webpackVersion = myVersion;
     environment.raven_dsn = config.raven_dsn_env_map[env.ENV] || (envName == 'dev' ? 'https://thecatiscute@not.in.dev/0' : 'no-dsn-found-in-config');
+    environment.nerd_count = '17,000';
```
Comment (rkeilty) [inline index.js]: We shouldn't make this a string, this should be an integer, and then the consuming applications should format as they see fit.

Comment (jeffbax) [inline index.js]: Also shouldn't this live within config.json, or get serialized from application?

Comment (rkeilty) [inline index.js]: We should just have this as:
    environment.nerd_count = 17000;
    environment.localized_nerd_count = parseInt(environent.nerd_count).toLocaleString();

File: index.js [hunk 2]
```diff
@@ -32,6 +32,8 @@ module.exports = function(options) {
     environment.webpackVersion = myVersion;
     environment.raven_dsn = config.raven_dsn_env_map[env.ENV] || (envName == 'dev' ? 'https://thecatiscute@not.in.dev/0' : 'no-dsn-found-in-config');
 
+    environment.nerd_count = 17000;
```
Comment (jeffbax) [inline index.js]: does this need a copy of the string as well?

**Pull request merged**